db/migrate/20181125004524_create_executions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateExecutions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :executions do |t|
+      t.integer :script_id, null: false, index: true
+      t.integer :server_id, null: false
+      t.string :status, limit: 20, default: Execution.statuses[:pending]
```

Comment:
El comentario de los 20 lo debería haber puesto acá, para reivindicarme sale una sugerencia:

```suggestion
      t.string :status, default: Execution.statuses[:pending]
```